### DISCUSSION TITLE
Update release version

### DISCUSSION THREAD

**Discussion by karkazzul** [dani-garcia/vaultwarden#3345] [Q&A]
Hello, I am currently using valutwarden in 1.27 version with the web version 2022.12.0. 
But, in the commmit for I see that the last version of web vault is 2023.2.0 [link to the github commit](https://github.com/dani-garcia/vaultwarden/pull/3281) but there is not new release since 24 december 2022. It's a bug or it's normal ? 

Thanks 
Karkazzul

**Comment by BlackDex** [comment]
If there would have been a new release it would be visible.
There are several changes which we needed to do for the new web-vault, which takes time.

**Reply by karkazzul** [reply]
okay, I better understand !
Thanks for you work, I love so much vaultwarden !
